science/imager/files/greg.in

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+export GAG_GAG="${HOME}/.gag"
+export GAG_ROOT_DIR="@PREFIX@/lib/imager"
+export GAG_PATH="${GAG_ROOT_DIR}/etc"
+export GAG_EXEC_SYSTEM="@ARCH@-darwin-gfortran"
+if [ -z "\$PYTHONPATH" ]; then
+  PYTHONPATH="${GAG_ROOT_DIR}/${GAG_EXEC_SYSTEM}/python"
+else
+  PYTHONPATH="${GAG_ROOT_DIR}/${GAG_EXEC_SYSTEM}/python:${PYTHONPATH}"
+fi
+export PYTHONPATH
+export DYLD_LIBRARY_PATH=${GAG_ROOT_DIR}/${GAG_EXEC_SYSTEM}/lib/
+exec ${GAG_ROOT_DIR}/${GAG_EXEC_SYSTEM}/bin/greg ${*}
```

Comment:
In all four scripts, this is the way to correctly handle arguments that might contain spaces or other special characters:

```suggestion
exec ${GAG_ROOT_DIR}/${GAG_EXEC_SYSTEM}/bin/greg "$@"
```

Reply:
Hello,
corrections done
thanks to recheck